src/partials/cart.hbs

```diff
@@ -0,0 +1,63 @@
+<div class="wrapper">
+  <!--Login Section-->
+  <div class="cart-section">
+    <div class="cart_header">
+      <h3>My Cart</h3>
```

Comment:
Why H3? We can't use H3 without parent H1>H2

Reply:
changed